CCS/C2000_C28x/README.md

```diff
@@ -0,0 +1,19 @@
+
+OVERVIEW
+
+This directory contains FreeRTOS port for Texas Instruments C28x based microcontrollers.
+
+The standard demo project to test this port is added at following location:
+FreeRTOS-Partner-Supported-Demos\C2000_F2838x_C28x_CCS\freertos_ex1_c28x_port_val
```

Comment:
```suggestion
FreeRTOS-Community-Supported-Demos\C2000_F2838x_C28x_CCS\freertos_ex1_c28x_port_val
```

Reply:
Updated the readme with the suggested change.